### PR TITLE
Update 004_70-Lava.txt

### PR DESCRIPTION
**Pull request by shoc71**
- 6 levels added with a lil side story on the side

### CODE REVIEW

File: Keep On Going/assets/levels/004_70-Lava.txt [hunk 1]
```diff
@@ -1691,3 +1691,847 @@ self.respawn_zones =  [
...
+        pygame.draw.rect(screen, (47, 237, 237), [1037, 230, 8, 58])
+]
+
+self.respawn_zones =  [
+        pygame.draw.rect(screen, (50, 205, 50), [35, 230, 8, 58])
+]
+
+======================================
+
+Red_Secret_Message (34, 320, 1)
```
Comment (slysnivy) [inline Keep On Going/assets/levels/004_70-Lava.txt:1806]: I'm pretty sure the space here won't interfere with anything but it should be fixed to stick with what we have.

File: Keep On Going/assets/levels/004_70-Lava.txt [hunk 2]
```diff
@@ -1691,3 +1691,847 @@ self.respawn_zones =  [
...
+self.win_zones =  [
+        pygame.draw.rect(screen, (47, 237, 237), [1041, 294, 8, 71])
+]
+
+self.respawn_zones =  [
+]
+
+===================================
+
+Red_volcano_smoking (333, 326, 1)
```
Comment (slysnivy) [inline Keep On Going/assets/levels/004_70-Lava.txt:2019]: Same spacing issue here

File: Keep On Going/assets/levels/004_70-Lava.txt [hunk 3]
```diff
@@ -1691,3 +1691,847 @@ self.respawn_zones =  [
...
+self.win_zones =  [
+        pygame.draw.rect(screen, (47, 237, 237), [1041, 294, 8, 71])
+]
+
+self.respawn_zones =  [
+]
+
+===================================
+
+Red_volcano_erupted (660, 326, 1)
```
Comment (slysnivy) [inline Keep On Going/assets/levels/004_70-Lava.txt:2165]: Space issue again

**Review by slysnivy — COMMENTED** [review]
Only minor issues with the spacing for level titles. These shouldn't conflict with how the program loads them, but for etiquette they could be fixed.

**Comment by shoc71** [comment]
thanks for pointing it out, I'll update the PR for with that in mind.

**Comment by shoc71** [comment]
done with a level made easier to beat.

**Pull request merged**